.github/workflows/yaml-lint.yml

```diff
@@ -3,10 +3,10 @@ name: YAML Lint
 on:
   push:
     branches:
-      - main
+      - '*'
   pull_request:
     branches:
-      - main
+      - '*'
```

Comment:
will this work?

```
on:
  push:
  pull_request:
```

Reply:
Yes, but the syntax it is slightly different I think:

`on: [push, pull_request]`